lnwallet/channel.go

```diff
@@ -232,6 +237,9 @@ func (c *commitment) toChannelDelta() (*channeldb.ChannelDelta, error) {
 	}
 
 	for _, htlc := range c.outgoingHTLCs {
+		if htlc.isDust {
```

Comment:
This was changed to include `isDustLocal` and `isDustRemote` instead, and those are used to avoid calling `locateOutputIndex`, since that would be a case where no output in the commit tx would be found.

It is also used in `fetchCommitmentView()` in order to avoid adding these HTLCs to the commit tx.